content/05.proteolysis.md

```diff
@@ -8,13 +8,65 @@ Peptides that are too short are less likely to uniquely match to a single protei
+Hence to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used in parallel to complement tryspin digestion [@PMID:23017020; @PMID:16512685]. 
+7. Glu-C:
+Glutamyl peptidase I, commonly known as Glu-C or V8 protease, is a serine protease obtained from _Staphyloccous aureus_.
+This specifically cleaves at the C-terimus of Glu. 
+However, depending on the pH conditions, in addition to Glu (pH 4.0) it is invloved in cleaving Asp (pH 8.0) residues also [@PMID:4627743].
+8. Asp-N:
+Peptidyl-Asp metallopeptidase, commonly known as Asp-N, is a metalloprotease obtained from _Pseudomonas fragi_ [@PMID:2669754].
+It is involved in the hydrolysis of Asp residues at the N-terminal of the peptide and works at an optimum pH 4-9.  
+Studies also suggests that Asp-N cleaves at the amino terminus of Glu when a detergent is present in the proteolysis buffer [@PMID:2669754]. 
+However, it efficiently cleaves at the N termini of Asp than Glu yielding many missed cleavages [@PMID:27123950].
+9. Alpha-lytic protease [@DOI:https://doi.org/10.1074/mcp.m113.034710] and how it enables mapping human SUMO sites [@PMID:29079793].
+10. Chymotrypsin:
+Chymotrypsin or Chymotrypsinogen A is a serine protease obtained from porcine or bovine pancreas and works at an optimum pH ranging from 7.8 to 8.0. 
+It cleaves at the C-terminus of hydrphobic amino acids Phe, Trp, Tyr and barely Met and Leu residues.
+Since the transmembrane region of membrance proteins lacks tryptic cleavage sites, this enzyme works well with membrane proteins having more hydrophobic residues [@PMID: 24870543; @PMID:24696503; @PMID:27123950].
+Also, the chymotryptic peptides generated after proteolysis will cover the proteome space orthogonal to that of tryptic peptides both in a quantitative and qualitative manner [@PMID:24290761; @PMID:22669647; @PMID:24696503]
+11. Arg-C:
+Clostripain, commonly known as Arg-C, is a cysteine protease obtained from _Clostridium histolyticum_.
+It hydrolyse mostly the C-terminal Arg residues and sometimes Lys residues, but with less efficiency.
+The peptides generated are generally longer than that of tryptic peptides.
+Arg-C is often used with other proteases for improving qualitative proteome data and also for investigating PTMs [@PMID:25823410]. 
+12. LysargiNase:
+LysargiNase, also known as Ulilysin, is a recently discovered protease belonging to the metalloprotease family. 
+It is a thermophilic protease derived from _Methanosarcina acetivorans_ that specifically cleaves at the N-terminus of Lys and Arg residues [@PMID:25419962].
+Hence, it enabled in the determination of C-terminal peptides, which otherwise was not possible using trypsin.
+In addition, it also favours the hydr0lysis of modified amino acids such as dimethylated or methylated arginine and lysine [@PMID:25419962].
+13. Lys-N:
+Peptidyl-Lys metalloendopeptidase, or Lys-N, is an metalloprotease obtained from _Grifola frondosa_. 
```

Comment:
citation?

Reply:
updated with a citation

Reply:
the citation doesn't appear to have made it into this line?

Reply:
I have put all the citations and updated the file. Dont know how all these citations didnt appear. However, I have again added all the references. 
[@PMID:19195997]